lib/matplotlib/path.py

```diff
@@ -166,8 +166,8 @@ def _fast_from_codes_and_verts(cls, verts, codes, internals_from=None):
 
         Parameters
         ----------
-        verts : numpy array
-        codes : numpy array
+        verts : array
```

Comment:
These go through `np.asarray`, so I assume array-like are fine.

Reply:
codes does not, but verts does, adjusted accordingly.